app/views/email_templates/index.html.slim

```diff
@@ -0,0 +1,22 @@
+h1 Alle E-mailvorlagen
+
+= form_navigation_btn :new
+
+.table-responsive
+  table.table
+    thead
+      th Betreff
+      th Textkörper
```

Comment:
"Textkörper" sounds very technical, "Text" is probably enough :)

(also in other places)